--- ./app/src/main/java/com/kintsugirun/WorkoutEditorScreen.kt
+++ ./app/src/main/java/com/kintsugirun/WorkoutEditorScreen.kt
@@ -23,10 +23,10 @@
     val workout by viewModel.workout.collectAsState()

     Scaffold(
-        topAppBar = {
+        topBar = {
             TopAppBar(
                 title = {
-                    val titleText = if (workout?.workoutName == "New Workout") "New Workout" else "Edit Workout"
+                    val titleText = if (workout?.workoutName == "New Workout") "New Workout" else "Edit Workout"
                     Text(titleText)
                 },
                 actions = {
